db/seeds/emancipation_data.rb

```diff
@@ -1,87 +1,87 @@
 # Emancipation Checklist Form Data
-category_housing = EmancipationCategory.create(name: "Youth has housing.", mutually_exclusive: true)
+category_housing = EmancipationCategory.where(name: "Youth has housing.").first_or_create(mutually_exclusive: true)
```

Comment:
While you're editing stuff, I think we wanted mutually exclusive to be false for housing.

Reply:
If i'm interpreting this correctly ![image](https://user-images.githubusercontent.com/8918762/101708321-9e26bc00-3a52-11eb-97cb-0d15babf829b.png)

Reply:
@compwron can a user have more than one housing option selected?

Reply:
@compwron I set the housing to mutually exclusive -- if only one option should be selected, then please flip it back to true before merging.

Reply:
For now, what we have is great! Zoe is writing a new github issue for complexities we discovered today. 

Long term, we need to allow let's assume that multiple housing options can be selected at once, *and* that new options which were once selected can be de-selected. For example, a teenage youth who is not getting along with their foster family might spend half the week at a friend's place, and half the week at their foster family's place. And then later no longer be with their foster family, and not able to stay with their friends, and be staying with grandparents instead. So we are not yet sure about how do to the UI for that complicated situation.